src/Testing/MockStatus.php

```diff
@@ -39,8 +39,8 @@ class MockStatus
     /** @var Code $code */
     public $code;
     public $details;
-    public $metadata;
-    public function __construct($code, $details = null, $metadata = null)
+    public array $metadata;
```

Comment:
Hey folks @bshaffer @noahdietz @PierrickVoulet 👋 - typed properties are only available in PHP >= 7.4.
Reverting this declaration will help https://github.com/googleapis/gax-php/runs/4158209307 pass

```suggestion
    public $metadata;
```

Reply:
Thanks @lucasmichot! Fixed.